lib/src/showcase.dart

```diff
@@ -98,6 +99,7 @@ class Showcase extends StatefulWidget {
     this.radius,
     this.onTargetLongPress,
     this.onTargetDoubleTap,
+    this.tipBorderRadius
```

Comment:
add comma at the end of paramter for better code structure. it will solve the code format error as well.